docs/integrations/adding_new.rst

```diff
@@ -11,6 +11,29 @@ The main points are:
 2. Find a place to enter and exit request scope and how to pass the container to a handler. Usually, it is entered in a middleware and container is stored in some kind of request context. 
 
    Alternatively, you can use the ``wrap_injection`` function with ``manage_scope=True`` to automate entering and exiting the request scope without relying on middleware. When enabled, ``manage_scope`` ensures that the container passed to ``wrap_injection`` enters and exits the next scope.
+
+   When you use ``manage_scope=True``, you can pass ``provide_context`` function that allows to populate the container context with passed arguments. This is useful when you want to pass the function scoped arguments to other dependencies without relying on middleware.
+
+   .. code-block:: python
+
+       from dishka import wrap_injection, Provider, provide
+
+       ...
+       class AppProvider(Provider):
+           request = from_context(provides=Request, scope=Scope.REQUEST)
+
+          @provide(scope=Scope.REQUEST)
+          def request(self, request: FromDishka[Request]) -> Iterable[User]:
+              yield get_user_from_request(request)
+
+       ...
+       wrapped_func = wrap_injection(
+           func=func,
+           is_async=False,
+           manage_scope=True,
+           provide_context=lambda req: Request: {Request: req},
```

Comment:
is it?

```python
provide_context(args, kwargs)
```

Reply:
As was discussed in tg, I've removed example section because it's not obvious